src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
This interface and its methods should have header comments. 
If not, developers who need to implement this in their class will not know what the expected behavior is.